code/backtracking/src/sudoku_solve/sudoku_solveNxN.cpp

```diff
@@ -0,0 +1,96 @@
+
+#include<bits/stdc++.h>
+
+using namespace std;
+
+const int n = 9;
+
+int grid[n][n];
+
+void input_grid() {
+    int i, j;
+    for (i = 0; i < n; i++) {
+        for (j = 0; j < n; j++) {
+            cin >> grid[i][j];
+        }
+    }
+}
+
+void print_grid() {
+    int i, j;
+    for (i = 0; i < n; i++) {
+        for (j = 0; j < n; j++) {
+            cout << grid[i][j] << " ";
+        }
+        cout << '\n';
+    }
+}
+
+bool can_be_placed(int row, int col, int num) {
+    int ii, jj;
+    ii = 0, jj = col;
```

Comment:
Declaration and assignment can happen on the same lines.

Reply:
It's just to show users so they can get confusion

Reply:
I think this:

```cpp
int ii = 0;
int jj = col;
```

will be clearer than both of these.

Reply:
Will do